libs/cypress-indexeddb/README.md

```diff
@@ -0,0 +1,126 @@
+<h1 align="center">Cypress IndexedDB helpers ⚙️</h1>
+
+[Cypress](https://cypress.io) IndexedDB helpers are a set of custom cypress commands that helps you handle indexedDB related operations in your Cypress tests.
+
+It supports:
+
+✅ &nbsp;Creating new IndexedDB databases and Object Stores <br/>
+✅ &nbsp;Making CRUD operations on the above-mentioned stores <br/>
+✅ &nbsp;Read data out of indexedDB and assert the results <br/>
+
+---
+
+<p align="center">
+  <a href="https://www.npmjs.com/package/@this-dot/cypress-indexeddb"><img src="https://img.shields.io/badge/%40this--dot-%2Froute--config-blueviolet" /></a>
+  <a href="https://www.npmjs.com/package/@this-dot/cypress-indexeddb"><img src="https://img.shields.io/npm/v/@this-dot/cypress-indexeddb" /></a>
+  <a href="https://github.com/thisdot/open-source/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://github.com/thisdot/open-source/actions/workflows/ci.yml/badge.svg" /></a>
+  <a href="https://github.com/thisdot/open-source/blob/main/LICENSE.md"><img src="https://img.shields.io/npm/l/@this-dot/cypress-indexeddb" /></a>
+  <a href="https://github.com/thisdot/open-source/issues"><img src="https://img.shields.io/github/issues/thisdot/open-source" /></a>
+</p>
+
+---
+
+## Usage
+
+### Installation
+
+1. Install the package:  
+   `npm install @this-dot/cypress-indexeddb`  
+   or  
+   `yarn install @this-dot/cypress-indexeddb`
+
+2. Import the plugin in your `cypress/support/commands.js` or `cypress/support/commands.ts` file:
+
+   ```typescript
+   import '@this-dot/cypress-indexeddb';
+   ```
+
+3. If you use typescript, set up typings in the same `support` folder
+   1. Create a `cypress-indexeddb-namespace.ts` file
+   2. Copy the following typings into the newly created file:
+      ```typescript
+      declare namespace Cypress {
+        interface Chainable<Subject> {
+          clearIndexedDb(databaseName: string): void;
+          openIndexedDb(databaseName: string, version?: number): Chainable<IDBDatabase>;
+          createObjectStore(storeName: string): Chainable<IDBObjectStore>;
+          getStore(storeName: string): Chainable<IDBObjectStore>;
+          createItem(key: string, value: unknown): Chainable<IDBObjectStore>;
+          readItem<T = unknown>(key: IDBValidKey | IDBKeyRange): Chainable<T>;
+          updateItem(key: string, value: unknown): Chainable<IDBObjectStore>;
+          deleteItem(key: string): Chainable<IDBObjectStore>;
+        }
+      }
```

Comment:
wouldn't it be possible to export this from the library itself? So that we can just add extends here? Something like
```suggestion
      declare namespace Cypress {
        interface Chainable<Subject> extends CypressIndexedDBHelpers { }
      }
```

Reply:
An issue for later to solve. Would love it though.